debugger/debugger.h

```diff
@@ -27,13 +27,16 @@ typedef enum { BYTE_STRIDE, WORD_STRIDE, DWORD_STRIDE } Stride;
 
 enum { MAX_BREAKPOINTS = 100 };
 
+enum {ERROR_ILLEGAL_INSTRUCTION = 1};
```

Comment:
spaces? (like in enum above, for consitency)

Reply:
Separate lines?

Reply:
Spaces, for consistency, OK.